Quaver.Shared/Screens/Gameplay/Rulesets/Input/InputManagerKeys.cs

```diff
@@ -199,7 +199,7 @@ private void HandleKeyPress(HitObjectManagerKeys manager, GameplayHitObjectKeys
             {
                 playfield.Stage.ComboDisplay.MakeVisible();
                 playfield.Stage.HitError.AddJudgement(judgement, gameplayHitObject.Info.StartTime - time);
-                playfield.Stage.JudgementHitBurst.PerformJudgementAnimation(judgement);
+                playfield.Stage.JudgementHitBurst[Math.Clamp(lane, 0, playfield.Stage.JudgementHitBurst.Count - 1)].PerformJudgementAnimation(judgement);
```

Comment:
Make a variable for `Math.Clamp(lane, 0, playfield.Stage.JudgementHitBurst.Count - 1)` because it's repeated very often